task-manager/main.py

```diff
@@ -4,6 +4,21 @@
 from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
 from sqlalchemy.future import select
 from models import AutoModels
+from pydantic import BaseModel # Added import for Form
+from typing import List # Added import for Form
+
+# Added for creating Film
+class FilmCreate(BaseModel):
```

Comment:
Ah, good approach here. Using Pydantic is smart. You must've read the documentation. :)